ref/rol_Java11.adoc

```diff
@@ -26,7 +26,7 @@ For Windows and Linux systems, you can use either the JDK from Oracle or the Jav
 
 == Java SE 11
 
-There are several Java 11 changes that impact Open Liberty. Between Java 8 and Java 11, several good enhancements such as local variables and the Flow API have been provided. However, there are also a few changes that are likely to cause problems for existing applications.
+There are several Java 11 changes that impact Open Liberty. Between Java 8 and Java 11, several good enhancements such as local variables and the Flow API have been provided. There are also a few changes that are likely to cause problems for existing applications.
```

Comment:
Suggestion:

Run the Acrolinx desktop checker on content before requesting a review. This is important. See my suggestions, some of which Acrolinx should flag:

Several Java SE Development Kit 11 changes impact Open Liberty. Between Java 8 and Java 11, several good enhancements such as local variables and the Flow API have been provided. There are also a few changes that are likely to cause problems for existing applications.